distributed/semaphore.py

```diff
@@ -172,6 +188,16 @@ async def acquire(self, comm=None, name=None, timeout=None, lease_id=None):
                     result,
                     w.elapsed(),
                 )
+
+                time_to_acquire_lease = (
+                    time() - self.pending_leases[name][pending_lease_id]
```

Comment:
I'd like to keep the code instrumentation as close as possible to the actual code it is supposed to observe. In particular this timing is mostly relevant in connection to the timeouts and the timeouts use the watch `w`. I'd prefer if we dont introduce another time measurement and just use `w.elapsed()` as for the log above

Reply:
- [ ] substitute this for `w.elapsed()`